sdk/canton/community/base/src/main/protobuf/com/digitalasset/canton/protocol/v4/participant_transaction.proto

```diff
@@ -12,14 +12,36 @@ import "com/digitalasset/canton/protocol/v2/participant_transaction.proto";
 import "com/digitalasset/canton/protocol/v3/participant_transaction.proto";
 import "scalapb/scalapb.proto";
 
-// Compared to v3: action_description changed type from v2.ActionDescription to v3.ActionDescription
+// Compared to v3: action_description changed type from v2.ActionDescription to v4.ActionDescription
```

Comment:
Ah, I see Simon added the v4 in a PR where he changes both v3 and adds v4. I'll ask him when he returns

Reply:
I checked `object ViewParticipantData.supportedProtoVersions`.

Reply:
Right right, So to follow the trend correctly, I would leave the new definition of FetchActionDescription in V4, and leave its ViewParticipantData unchanged, then make a V5 that just defines the ViewParticipantData using the new Action description from V4

Reply:
@rgugliel-da Samual needs to define a new `message FetchActionDescription`, and therefore also `message ActionDescription` and `message ViewParticipantData`. The current head version numbers are:

- FetchActionDescription: 1
- ActionDescription: 3
- ViewParticipantData: 4

Normally, the new versions would be 2/4/5. However, Samual wants to make them 4/4/5, because `FetchActionDescription` is defined inside of `ActionDescription`. Please tell us which version numbers to use.

Reply:
I would use 2/4/5: we allow "inconsistent nesting" in our messages

Reply:
Yep, that's fine :)

Reply:
I've pushed up a commit to fix this